db/migrate/20161014133811_move_content_view_version_description_to_histories.rb

```diff
@@ -25,7 +25,8 @@ def up
       publish_history ||= CVHistory.create!(action: CVHistory.actions[:publish],
                                             katello_content_view_version_id: version.id,
                                             status: 'successful',
-                                            user: ''
+                                            user: '',
+                                            created_at: version.created_at
```

Comment:
i don't know that modifying the old migration is useful at this point, its over 1.5 years old.    we'd want a new migration to fix this if possible

Reply:
Somehow, this migration was creating records from 6.2 to 6.3 upgrade (I think, see BZ/case), and those records had a `DateTime.now` `created_at` date. I guess we could make a new migration, but it'd be impossible to know if we should update the created_at date from the version or not.

Reply:
I guess this kind of change is useless but not harmful in master - It's meant mainly as a patch to cherry pick into prior releases.

Reply:
yeah, its fine to include as long as its not required for this fix to work.